content/docs/guides/api-automation.md

```diff
@@ -34,7 +34,7 @@ For more information see the [Karate project on GitHub](https://github.com/intui
 # API Automation Tools for Ruby
 
 ## REST Client
-[REST Client](https://github.com/rest-client/rest-client) is a simple HTTP and REST client for Ruby.
+[REST Client](https://github.com/rest-client/rest-client) is an HTTP and REST client for Ruby.
```

Comment:
I'm not against this, but this definition (simply HTTP and REST client), is used all over the web as their official title.

Reply:
Personally I don't love their official title 😂 but if it's more familiar to people then I can add it back.